File: 01_Basico/04 - Funcoes_Modulos/Resposta_em_Kotlin/questao03.kt
// Forma 01, forma simples e feita na mão
fun verificar_frase(frase: String, letra: String): Int{
    var contador = 0
    var tamanho = frase.length
    for (i in frase) {
        if (letra == i) {
            contador++
        }
    }
    return "A letra '${letra}' aparece na frase '${frase}', um total de ${contador}"
}

//Aqui estamos recebendo os valores de a e b que serão o posteriormente os valores dos parametros da função

println("Digite uma frase: )
let a = readLine()!!
println("Digite uma letra: )
let b = readLine()!!

//Aqui chamamos a função, perceba que os atributos dos parametros "frase" e "letra" são respectivamente "a" e "b"
var primeiro_teste = verificar_frase(a, b)

println(primeiro_teste)

File: 01_Basico/04 - Funcoes_Modulos/Resposta_em_JavaScript/questao03.js
// Forma 01, forma simples e feita na mão
function verificar_frase(frase, letra){
    var contador = 0
    let tamanho = frase.length
    for (i = 0; i < tamanho ; i++){
        if (letra == frase[i]) {
            contador++
        }
    }
    return `A letra '${letra}' aparece na frase '${frase}', um total de ${contador}`
}

//Aqui estamos recebendo os valores de a e b que serão o posteriormente os valores dos parametros da função

let a = prompt("Digite uma frase: ")
let b = prompt("Digite uma letra: ")

//Aqui chamamos a função, perceba que os atributos dos parametros "frase" e "letra" são respectivamente "a" e "b"
let primeiro_teste = verificar_frase(a, b)

console.log(primeiro_teste)
